riscv/v_ext_macros.h

```diff
@@ -1202,10 +1202,10 @@ reg_t index[P.VU.vlmax]; \
 
 #define VI_LD(stride, offset, elt_width, is_mask_ldst) \
   const reg_t nf = insn.v_nf() + 1; \
+  VI_CHECK_LOAD(elt_width, is_mask_ldst); \
   const reg_t vl = is_mask_ldst ? ((P.VU.vl->read() + 7) / 8) : P.VU.vl->read(); \
   const reg_t baseAddr = RS1; \
   const reg_t vd = insn.rd(); \
-  VI_CHECK_LOAD(elt_width, is_mask_ldst); \
```

Comment:
Don't you need to make the analogous change here?  https://github.com/riscv-software-src/riscv-isa-sim/blob/cc8c2fc923c62132f208d29af745b2db4aecc75a/riscv/v_ext_macros.h#L1313

Reply:
And here? https://github.com/riscv-software-src/riscv-isa-sim/blob/cc8c2fc923c62132f208d29af745b2db4aecc75a/riscv/v_ext_macros.h#L1223

Reply:
And here? https://github.com/riscv-software-src/riscv-isa-sim/blob/cc8c2fc923c62132f208d29af745b2db4aecc75a/riscv/v_ext_macros.h#L1277

Reply:
Update PR accordingly.